ajax/libs/vjs-video/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "vjs-video",
+  "version": "0.1.10",
+  "description": "An angular js directive for video.js",
+  "homepage": "http://lonnygomes.github.io/vjs-video/",
```

Comment:
@nagrawal3 It will be better to use `https` instead of `http` for __github.io__ domain.